zha/application/platforms/sensor/__init__.py

```diff
@@ -733,13 +724,27 @@ def _divisor(self, value: int | float | None) -> None:
         raise AttributeError("Cannot set divisor directly")
 
 
+# this entity will be created by ReportingEM or PolledEM class below
+class ElectricalMeasurementActivePower(BaseElectricalMeasurement):
+    """Active power phase measurement."""
+
+    _attribute_name = "active_power"
+    # no unique id suffix for backwards compatibility
+    # no translation key due to device class
+    _attr_max_attribute_name = "active_power_max"
+    _divisor_attribute_name = "ac_power_divisor"
+    _multiplier_attribute_name = "ac_power_multiplier"
+    _attr_device_class: SensorDeviceClass = SensorDeviceClass.POWER
+    _attr_native_unit_of_measurement: str = UnitOfPower.WATT
```

Comment:
Explanation:
Previously, `PolledElectricalMeasurement` was the base class for everything, yet also the `active_power` sensor.

Due to how polling works for the EM sensors, we now have a `ElectricalMeasurementActivePower` class that does nothing on its own.
The entity is only created by either `ReportingElectricalMeasurement` (which does no polling) or by `PolledElectricalMeasurement`, which then polls all relevant EM attributes as before (using `async_update` in the EM cluster handler).

This generally allows for a much cleaner class hierarchy.